src/backend/access/transam/xlogfuncs.c

```diff
@@ -23,6 +23,9 @@
 #include "access/xlogutils.h"
 #include "catalog/catalog.h"
 #include "catalog/pg_type.h"
+#include "cdb/cdbvars.h"
+#include "cdb/cdbdisp_query.h"
+#include "cdb/cdbdispatchresult.h"
```

Comment:
Need to move these header includes to the bottom (after `#include "storage/fd.h"`) with a newline separator to reduce merge conflicts.